### ISSUE TITLE
farm/cli --mode only surpport development or production

### ISSUE THREAD

**Issue by sepZeng** [farm-fe/farm#1160]
### Steps to reproduce

  "test": "farm build --mode test",

### What is expected?

it could support

### What is actually happening?

thread '<unnamed>' panicked at crates\node\src\lib.rs:176:8:
can not transform js config object to rust config: Error { status: InvalidArg, reason: "unknown variant `test`, expected `development` or `production`", maybe_raw: 0x0 }

### System Info

```Shell
System:
    OS: Windows 10 10.0.19045
    CPU: (8) x64 AMD Ryzen 5 3550H with Radeon Vega Mobile Gfx  
    Memory: 6.41 GB / 14.88 GB
  Binaries:
    Node: 20.12.0 - C:\Program Files\nodejs\node.EXE
    npm: 10.5.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Chromium (123.0.2420.81)
  npmPackages:
    @farmfe/core: ^1.0.0 => 1.0.16
    @farmfe/cli:^1.0.0
```

### Any additional comments?

no

**Comment by ErKeLost** [comment]
Have you installed the latest version of farm?

**Comment by sepZeng** [comment]
> Have you installed the latest version of farm?

i'm sure,i update dependcy every three days，i just now npm create farm@lastet(add --mode test),this project is still compiled failed.

**Comment by ErKeLost** [comment]
There is a problem. i will check it,  thanks reports

**Comment by sepZeng** [comment]
> > Have you installed the latest version of farm?
> 
> i'm sure,i update dependcy every three days，i just now npm create farm@lastet(add --mode test),this project is still compiled failed.

i update farmfe/core   to  1.0.17 ,issue still exist

**Comment by ErKeLost** [comment]
@sepZeng  fixed in 1.0.18
